proposals/records-wip.md

```diff
@@ -28,3 +28,17 @@ struct_body
 ```
 
 The `attributes` non-terminal will also permit a new contextual attribute, `data`.
+
+A class (struct) declared with the a parameter list is called a record class (record struct), either of which is a record type.
+
+## Members of a record type
+
+In addition to the members declared in the class-body, a record type has the following additional members:
+
+**Primary Constructor**
+
+A record type has a public constructor whose signature corresponds to the value parameters of the
+type declaration. This is called the primary constructor for the type, and causes the implicitly
+declared default constructor to be suppressed. If a constructor with the same signature is
+already present in the class, either through inheritance or user declaration, the synthesized
+primary constructor is suppressed.
```

Comment:
If it is suppressed, where is the code to do the things the implicitly generated constructor would do?